src/client_side.cc

```diff
@@ -2176,35 +2176,11 @@ clientLifetimeTimeout(const CommTimeoutCbParams &io)
 
 ConnStateData::ConnStateData(const MasterXaction::Pointer &xact) :
     AsyncJob("ConnStateData"), // kids overwrite
-    Server(xact),
-    bodyParser(nullptr),
+    Server(xact)
 #if USE_OPENSSL
-    sslBumpMode(Ssl::bumpEnd),
-    tlsParser(Security::HandshakeParser::fromClient),
+    , tlsParser(Security::HandshakeParser::fromClient)
```

Comment:
If possible, please move this initialization into the header file as well.

Reply:
Not possible without adding move semantics. This is not a trivial C++11 assignment, but a call to a specific sub-constructor of the Parser class.